server/utils/helpers/getCurrentPay.test.ts

```diff
@@ -0,0 +1,107 @@
+import { Activity, Allocation, PrisonPayBand } from '../../@types/activitiesAPI/types'
+import { Prisoner } from '../../@types/prisonerOffenderSearchImport/types'
+import getCurrentPay from './getCurrentPay'
+
+const prisoner = {
+  prisonerNumber: 'G0995GW',
+  currentIncentive: {
+    level: { code: 'STD', description: 'Standard' },
+    dateTime: '2017-01-12T12:08:12',
+    nextReviewDate: '2018-01-12',
+  },
+} as unknown as Prisoner
+
```

Comment:
Where possible I prefer strong typing rather than 'as', but appreciate there is a lot of extra parameters to set here, so probably not worth it here

Reply:
Completely agree - I try to avoid it where possible but this is quite a big object with data we don't need for the test